docs/codeflow/codeflow-faq.md

```diff
@@ -1,11 +1,7 @@
 ---
-title: &title Codeflow FAQ
-description: &description 'This page addresses the frequently asked questions concerning: current features, security, planned features, pricing and access.'
-head:
-  - ['meta', {property: 'og:title', content: *title}] 
-  - ['meta', {property: 'og:image', content: 'https://developer.stackblitz.com/img/og/codeflow-faq.png'}]
-  - ['meta', {name: 'twitter:title', content: *title}]
-  - ['meta', {name: 'twitter:description', content: *description}]
+title: Codeflow FAQ
+description: 'This page addresses the frequently asked questions concerning: current features, security, planned features, pricing and access.'
+og_image: codeflow-faq.png
```

Comment:
We have a few possible options for how to define the `og:image` value in each page:

1. Provide the file name only, and build the full URL using a common prefix in JS (what I opted for here).
2. Provide a local path from the roots of the `docs/public` folder, such as `img/og/codeflow-faq.png`.
3. Provide the full URL, such as `https://developer.stackblitz.com/img/og/codeflow-faq.png`.

I used the first option because it will be easier to handle changes if we end up moving the `docs/public/img/og` folder somewhere else and/or if we serve the docs site on a subpath of stackblitz.com.